crypto/lhash/lh_stats.c

```diff
@@ -92,6 +96,7 @@ void OPENSSL_LH_stats_bio(const OPENSSL_LHASH *lh, BIO *out)
         CRYPTO_THREAD_unlock(lh->tsan_lock);
 #endif
```

Comment:
Nit: Needs an extra space `# endif`, (and in the corresponding opengin `ifdef` that github won't let me comment on.